protobufts/descriptor.proto

```diff
@@ -0,0 +1,909 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Are these modified at all? If not, can we not pull them in directly via bazel rather than adding to source?

Reply:
Like how we do this for e.g. `annotations.proto`

Reply:
nice suggestion, cleaned up a lot!